mlflow/pyspark/ml/log_model_allowlist.txt

```diff
@@ -0,0 +1,40 @@
+# classification
+pyspark.ml.classification.LinearSVC
+pyspark.ml.classification.DecisionTreeClassifier
+pyspark.ml.classification.GBTClassifier
+pyspark.ml.classification.RandomForestClassifier
+pyspark.ml.classification.NaiveBayes
+
+# clustering
+pyspark.ml.clustering.BisectingKMeans
+pyspark.ml.clustering.KMeans
+pyspark.ml.clustering.GaussianMixture
+
+# Recommendation
+pyspark.ml.recommendation.ALS
+
+# Regression
+pyspark.ml.regression.AFTSurvivalRegression
+pyspark.ml.regression.DecisionTreeRegressor
+pyspark.ml.regression.GBTRegressor
+pyspark.ml.regression.GeneralizedLinearRegression
+pyspark.ml.regression.LinearRegression
+pyspark.ml.regression.RandomForestRegressor
+
+# featurizer
+pyspark.ml.feature.BucketedRandomProjectionLSH
+pyspark.ml.feature.ChiSqSelector
+pyspark.ml.feature.CountVectorizer
+pyspark.ml.feature.IDF
+pyspark.ml.feature.Imputer
+pyspark.ml.feature.MaxAbsScaler
+pyspark.ml.feature.MinHashLSH
+pyspark.ml.feature.MinMaxScaler
+pyspark.ml.feature.OneHotEncoder
```

Comment:
Should we avoid auto-logging featurizers? I think it's common to use featurizer and estimator separately like below? If we auto-log featurizers, the code below would create two runs.

```python
cv = CountVectorizer.fit(df)
df = cv.transform(df)

model = LinearRegression().fit(df)
preds = model.transform(df)
```